### PR TITLE
Update tools_to_keep

### PR DESCRIPTION
**Pull request by EngyNasr**
Adding missing tools found in [microGalaxy workflows and trainings](https://docs.google.com/spreadsheets/d/11Z1yXtS4GTER3QCyAIRqWupr5e43YFFKeMk8UyfYfj4/edit#gid=905989451)

### CODE REVIEW

File: data/communities/microgalaxy/tools_to_keep [hunk 1]
```diff
@@ -1,138 +1,143 @@
...
 fasttree
+featurecounts
+filter_spades_repeats
 filtlong
+flashlfq
+flye
+format_metaphlan2_output
 fraggenescan
+freebayes
+freebayes
```
Comment (EngyNasr) [inline data/communities/microgalaxy/tools_to_keep:77]: I will remove this line

File: data/communities/microgalaxy/tools_to_keep [hunk 2]
```diff
@@ -142,77 +147,124 @@ merqury
...
+metaquantome
 minia
 miniasm
+minimap2
+minipolish
 miniprot
 mitos
 mlst
+mob_suite
+mob_suite
```
Comment (EngyNasr) [inline data/communities/microgalaxy/tools_to_keep:163]: and this line too

File: data/communities/microgalaxy/tools_to_keep [hunk 3]
```diff
@@ -142,77 +147,124 @@ merqury
...
+snp-dists
+snp-sites
+snpsift
 sonneityping
+sortmerna
 spades
 spotyping
+sr_bowtie
 srst2
+srst2
```
Comment (EngyNasr) [inline data/communities/microgalaxy/tools_to_keep:245]: this too

**Comment by EngyNasr** [comment]
I also rearranged the table alphabetically, as the majority was arranged but few on top were not, but now the full table is arranged

**Comment by paulzierep** [comment]
Uhh, the tools_to_keep are currently fetched automatically:
https://github.com/galaxyproject/galaxy_tool_metadata_extractor/blob/cea51e857e8839cdd61dfe3dc61e065f084f7c18/bin/get_community_tools.sh#L10-L20

@bebatut should we stop this behaviour, in favour of @EngyNasr update, or update the google file ?

**Comment by bebatut** [comment]
Yes we should probably stop this

**Comment by paulzierep** [comment]
thank you @EngyNasr !

**Pull request merged**